repo/packages/M/marathon/0/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name" : "marathon",
+  "version" : "0.13.0-RC3",
+  "scm" : "https://github.com/mesosphere/marathon.git",
+  "maintainer" : "support@mesosphere.io",
+  "description" : "A cluster-wide init and control system for services in cgroups or Docker containers.",
+  "framework" : true,
+  "images" : {
+    "icon-small" : "https://downloads.mesosphere.io/marathon/assets/icon-service-marathon-small.png",
```

Comment:
This should use https://downloads.mesosphere.com. (Same for the other assets below.)